hack/boilerplate/boilerplate.py

```diff
@@ -146,6 +146,7 @@ def file_extension(filename):
     "test/e2e/load_balancer.go",
     "test/e2e/framework/cleanup.go",
     "test/e2e/framework/framework.go",
+    "test/e2e/framework/seclist_utils.go",
```

Comment:
seclist_utils => seclist_util to be consistent with test/e2e/framework/service_util.go

Reply:
It was matching actually file called 'network_utils'. I made all util files singular.